File: vortexdbg-android/src/main/java/com/vortexdbg/linux/android/dvm/api/ClassLoader.java
package com.vortexdbg.linux.android.dvm.api;

import com.vortexdbg.linux.android.dvm.DvmObject;
import com.vortexdbg.linux.android.dvm.VM;

public class ClassLoader extends DvmObject<String> {

    public ClassLoader(VM vm, String value) {
        super(vm.resolveClass("dalvik/system/PathClassLoader"), value);
    }

}

File: vortexdbg-android/src/main/kotlin/com/vortexdbg/linux/android/dvm/api/ClassLoader.kt
package com.vortexdbg.linux.android.dvm.api

import com.vortexdbg.linux.android.dvm.DvmObject
import com.vortexdbg.linux.android.dvm.VM

open class ClassLoader(vm: VM, value: String) : DvmObject<String>(vm.resolveClass("dalvik/system/PathClassLoader"), value)
